### PR TITLE
Update documentation for deploying custom driver

### PR DESCRIPTION
**Pull request by hsadoyan**
Add changelogs for latest patches

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

 /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Update documentation for deploying custom driver
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +151,23 @@ $ PROJECT=<your-gcp-project> GCFS_SA_DIR=<your-directory-to-store-credentials-by
   testing, and the `dev` overlay is for driver development. `./deploy/kubernetes/cluster-setup.sh`
   will install the driver pods, as well as necessary RBAC and resources.
 
+* If deploying new changes in the Update the overlay file with a new custom tag to identify this image.
```
Comment (leiyiz) [inline README.md:154]: pls double check this line

Comment (hsadoyan) [reply README.md]: Oops. Fixed.

**Review by leiyiz — COMMENTED** [review]
/lgtm
/approve
/ok-to-test

**Pull request merged**